### DISCUSSION TITLE
Dim screen lower than zero ?

### DISCUSSION THREAD

**Discussion by savi4587** [seerge/g-helper#2371] [Ideas]
Hi , is there a way you could put a setting to lower the brightness lower than the typical zero on Zephyrus laptops ? Mine is still quite bright at 0 at night. GU603VV

**Comment by seerge** [comment]
@savi4587 hello, no. Unfortunately 0 is as dim as you could get. But you can combine regular brightness adjustment in windows with dimming to make it even dimmer :)

**Comment by Bananalaw** [comment]
MyMonic - good utility for brightness controls. It also allow to control brightness trough gpu.
